### PR TITLE
feat: I modified some files

### LINKED ISSUE #20: create reusable section
and add the youtube button.

### PR DESCRIPTION
**Pull request by Alez05**
fix #20

### CODE REVIEW

**Review by florin-bizgan — APPROVED** [review]
🔝

**Review by flivanov92 — APPROVED** [review]
👍

File: src/_component/welcome/welcome.pug [hunk 1]
```diff
@@ -24,8 +24,18 @@ block awesome-content
     .header-wrapper-orange
       .header-orange= info
 
-    .box__info-box
-      .text__section-title= moreTitle
+    .flex-gap16
+      .box__gap-8
```
Comment (MariaTeo) [inline src/_component/welcome/welcome.pug:28]: ? why do you need this if there are no children in it?

**Review by MariaTeo — APPROVED** [review]

File: src/_component/welcome/welcome.pug (hunk 1 above)
Comment (MariaTeo) [reply src/_component/welcome/welcome.pug:28]: ?

Comment (MariaTeo) [inline src/_component/welcome/welcome.pug:28]: aicea

Comment (Alez05) [reply src/_component/welcome/welcome.pug:28]: let me check

File: src/_component/welcome/welcome.pug [hunk 2]
```diff
@@ -24,8 +24,18 @@ block awesome-content
...
+      .box__info-box
```
Comment (MariaTeo) [inline src/_component/welcome/welcome.pug:29]: linia 28

**Pull request merged**